kronolith/js/kronolith.js

```diff
@@ -5997,6 +5999,18 @@ KronolithCore = {
         $('kronolithEventLinkExport').up('li').show();
         $('kronolithEventExport').href = Kronolith.conf.URI_EVENT_EXPORT.interpolate({ id: ev.id, calendar: ev.c, type: ev.ty });
 
+        /* Attendance and Status */
+        if (!ev.oy) {
+            $('kronolithEventStatus').disabled = true;
+            $('kronolithEventAttend').show();
+        } else {
+            $('kronolithEventStatus').disabled = false;
+            $('kronolithEventAttend').hide();
+        }
+        if (ev.cr) {
+            $('kronolithEventAttendance').setValue(ev.cr);
```

Comment:
This doesn't seem to work. I didn't check but either chaning this field doesn't save the current reply status (in the attendee's copy), or the drop down isn't populated correctly.